auth.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Intensio</title>
+    <link rel="icon" href="res/logo.svg" type="image/svg">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&family=Sanchez&display=swap"
+        rel="stylesheet">
+    <link href="styles/shared.css" rel="stylesheet">
+    <link href="styles/auth.css" rel="stylesheet">
+</head>
+
+<body class="landing-page">
+    <header>
+        <a class="landing-logotype" href="index.html">
+            <img src="res/logo.svg" alt="">
+            <div>
+                <h1>intensio</h1>
+                <span>Master your time</span>
+            </div>
+        </a>
+    </header>
+    <main>
+        <span class="login-form-title">Start using <span>intensio</span></span>
+        <form method="post" class="login-form">
+
+            <label hidden for="email">Email</label>
+            <input class="text-input" type="email" id="email" name="email" placeholder="Email">
+
+            <label hidden for="password">Password</label>
```

Comment:
Why do we need a hidden label here?

Reply:
The main thought was that these labels would be visible for screen-readers, so it could help to navigate on the page